third_party/Makefile

```diff
@@ -86,12 +86,12 @@ ifneq (,$(wildcard fio/Makefile))
 	cd ./fio && ./configure --prefix=$(SB_MICRO_PATH) --disable-native && make -j && make install
 endif
 
-# Build rccl-tests from commit 2a18737 of default branch.
+# Build rccl-tests with develop branch.
 rocm_rccl_tests: sb_micro_path
-ifneq (, $(wildcard rccl-tests/Makefile))
-	cd ./rccl-tests && make MPI=1 MPI_HOME=$(MPI_HOME) HIP_HOME=$(HIP_HOME) RCCL_HOME=$(RCCL_HOME) -j
-	cp -v ./rccl-tests/build/* $(SB_MICRO_PATH)/bin/
-endif
+	if [ -d rccl-tests ]; then rm -rf rccl-tests ; fi
```

Comment:
Why do we remove rccl-tests from third party? Shall we update rccl submodule's commit ID instead?

Reply:
ok, revised